doc/source/whatsnew/v0.19.0.txt

```diff
@@ -229,6 +229,7 @@ Other enhancements
 - ``pd.read_html()`` has gained support for the ``decimal`` option (:issue:`12907`)
 - A top-level function :func:`union_categorical` has been added for combining categoricals, see :ref:`Unioning Categoricals<categorical.union>` (:issue:`13361`)
 - ``Series`` has gained the properties ``.is_monotonic``, ``.is_monotonic_increasing``, ``.is_monotonic_decreasing``, similar to ``Index`` (:issue:`13336`)
+- ``DataFrame.sort_values()`` has gained support for re-ordering columns by index label (:issue:`10806`)
```

Comment:
Don't they mean the same thing? (`by` is always required `sort_values`, and axis 1 is equivalent to "columns".)  See if the new wording in 2773cdf hits what you're getting at, otherwise can you please specify?